upcoming-release-notes/2362.md

```diff
@@ -0,0 +1,6 @@
+---
+category: Enhancements
+authors: [joewashear007]
+---
+
+Add option to authenticate with HTTP header from Auth Proxy. Set new env `ACTUAL_LOGIN_METHOD` = 'header' and make sure your proxy set the `X-ACTUAL-PASSWORD` to your actual password. This front end will attempt to auto login into actual if the header is accepted by the server. If the header is not valid or incorrect, the user is notified and given the option to use their password to login. Please also see the server PR: https://github.com/actualbudget/actual-server/pull/312
```

Comment:
Release note should be just the first sentence here.

Reply:
Done